recipes/opencv/4.x/conanfile.py

```diff
@@ -1127,7 +1127,7 @@ def requirements(self):
         if self.options.get_safe("with_webp"):
             self.requires("libwebp/1.3.2")
         if self.options.get_safe("with_gdal"):
-            self.requires("gdal/3.7.0")
+            self.requires("gdal/3.4.3")
```

Comment:
Is this actually a downgrade? Can you link/elaborate a bit on it? Thanks!

Reply:
This is because my bumping bot only knows versions which have conan 2 compatible recipes (gdal >3.5 is not yet)